sort.php

```diff
@@ -1,5 +1,25 @@
 <?php
 
+function printHelp()
+{
+    echo "Usage: php sort.php [OPTION] < [FILE]
```

Comment:
Please use so called heredocs for such strings.

Reply:
Still missing.

Reply:
check the second commit

Reply:
"Usage" was right. I asked you to use heredocs.